### ISSUE TITLE
Cannot find name 'require'

### ISSUE THREAD

**Issue by markgoho** [surmon-china/ngx-quill-editor#9]
```
ERROR in C:/Users/Mark/Documents/GitHub/quillTest/node_modules/ng2-quill-editor/quillEditor.component.ts (14,15): Cannot find name 'require'.)
C:/Users/Mark/Documents/GitHub/quillTest/node_modules/ng2-quill-editor/quillEditor.component.ts (23,5): Cannot find name 'require'.)
C:/Users/Mark/Documents/GitHub/quillTest/node_modules/ng2-quill-editor/quillEditor.component.ts (24,5): Cannot find name 'require'.)
C:/Users/Mark/Documents/GitHub/quillTest/node_modules/ng2-quill-editor/quillEditor.component.ts (25,5): Cannot find name 'require'.)
```

Any ideas what's happening here? I'm using Angular v4 Final

**Comment by musecz** [comment]
Just add this in quillEditor.component.ts : 
`declare var require: any;`

**Comment by surmon-china** [comment]
Update to v2.0.2 please, It should be work!

**Comment by sagarvd01** [comment]
@surmon-china  this issue occurs in the fresh installation I did today. Solved by using the method suggested by @schnouz I think you should add it into the code

**Comment by surmon-china** [comment]
@sagarvd01 https://github.com/surmon-china/ngx-quill-editor/blob/master/quillEditor.component.ts#L14

Thx!

**Comment by gesheftigg** [comment]
I made the fix suggested by @schnouz and it worked when I ran the project using "npm start". However, when I try to package the code for deploy using "ng build --prod" I get the following error

ERROR in Error encountered resolving symbol values statically. Reference to a local (non-exported) symbol 'require'. Consider exporting the symbol (position 14:13 in the original .ts file), resolving symbol QuillEditorComponent in /Users/<path to directory>/node_modules/ng2-quill-editor/quillEditor.component.ts

I am on a MAC
